src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionListener.java

```diff
@@ -12,28 +14,53 @@
 public abstract class FlowExecutionListener implements ExtensionPoint {
 
     /**
-     * Called when a {@link FlowExecution} has started running or resumed.
+     * Called when a {@link FlowExecution} has started running.
      *
-     * @param execution The {@link FlowExecution} that has started running or resumed.
-     * @param resumed True the execution is resuming, false if it's starting for the first time.
+     * The {@link FlowExecution} will already have been added to the {@link FlowExecutionList} by this point.
+     *
+     * @param execution The {@link FlowExecution} that has started running.
+     */
+    public void onRunning(FlowExecution execution) {
+    }
+
+    /**
+     * Called when a {@link FlowExecution} has resumed.
+     *
+     * The {@link FlowExecution} will already have been added to the {@link FlowExecutionList} by this point.
```

Comment:
Well, yes, but perhaps misleading because it has been there for a while—this list is persistent.